control/control.CASP.xml

```diff
@@ -314,44 +295,41 @@ Please visit us at http://www.suse.com/.
         <service_sfcb><label>CIM Server</label></service_sfcb>
         <roles_caption>
           <!-- TRANSLATORS: dialog caption -->
-          <label>System Role</label>
+          <label>CASP Roles</label>
```

Comment:
But both are system role. I think it is fine to look it similar, just with specialized content for CASP that are different to SLES. SLED do not have it yet, but if we want it for them, still I think we can keep system role as it fit into goal of that dialog, not?